14_functions_lab/01.py

```diff
@@ -1,7 +1,23 @@
-"""
-Write 2 functions:
-    mysum - returns the sum of its input arguments
-    mymul - returns the multiplication of its input arguments
-    Ignore non-numeric arguments
-"""
-
+
+import sys
+from operator import mul
+import argparse
+
+def sum_params(*nums):
+	return sum([n for n in nums if type(n)==int])

```

Comment:
Nice! Python also has the following code to check for any numeric value (int or float):

```
import numbers
isinstance(10, numbers.Number)
```